### DISCUSSION TITLE
rotate 180 / flip upside-down camera image

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#7832] [Ask A Question] [answered]
I have a Dahua PTZ1A225-HNR-XA camera. The problem is that the camera image is **upside down**. I am hoping someone here might know how to fix this.

I've tried to flip (or 180-degrees rotate) the image by changing the appropriate setting in the camera's GUI. Unfortunately, there doesn't seem to be a setting that I can change through the GUI. The closest seems to be "View Angle Mode" which is set to "Auto" in a drop-down menu with no other options.

I'm guessing there must be a way to flip/rotate the image using the Dahua HTTP API by issuing a ```curl``` command from the command line. The relevant setting seems to be VideoImageControl[0].Flip=true. But I think this is being overridden by the camera because the VideoImageControl[0].FlipMode=Auto. Unfortunately, I don't know what the other FlipMode settings are. I've tried 0, 1, 2, and Manual and Flip to no avail.

Any suggestions on what to do?

**Comment by NickM-27** [comment]
The option is there on my camera under conditions -> flip

![Screenshot_20230915_172707_Chrome Beta.jpg](https://github.com/blakeblackshear/frigate/assets/14866235/7c6a18d9-9842-400b-b5fa-24220633c103)

**Reply by Cold-Lemonade** [reply, marked as answer]
Thanks. But that option does not exist in my camera's GUI. But I found it... I had to select "inverted" under "installed mode" in the "General -> System" tab.
